SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -23,14 +23,32 @@ struct ConfigureConnection
 		: Config(InConfig)
 		, Params()
 		, WorkerType(*Config.WorkerType)
-		, ProtocolLogPrefix(*FormatProtocolPrefix())
+		, WorkerLogPrefix(*FormatLogFilePrefix())
 	{
 		Params = Worker_DefaultConnectionParameters();
 
 		Params.worker_type = WorkerType.Get();
 
-		Params.enable_protocol_logging_at_startup = Config.EnableProtocolLoggingAtStartup;
-		Params.protocol_logging.log_prefix = ProtocolLogPrefix.Get();
+		Logsink.logsink_type = WORKER_LOGSINK_TYPE_ROTATING_FILE;
+		Logsink.rotating_logfile_parameters.log_prefix = WorkerLogPrefix.Get();
+		Logsink.rotating_logfile_parameters.max_log_files = 1;
+		Logsink.rotating_logfile_parameters.max_log_file_size_bytes = 10485760; // 10 MB
+
+		uint32_t Categories = 0;
+		if (Config.EnableOpLogging)
+		{
+			Categories |= WORKER_LOG_CATEGORY_API;
```

Comment:
So wait, I need some more context. Does this mean that worker logs will now be forwarded to this file rather than the previous location? Worker logs are supposed to mean `Logs created by the worker` and are forwarded to the runtime via the `WorkerLogMessageHandler` or something. Is this concept replacing this idea or is this a different type of `WorkerLog`?

Additionally, if these logs are important they should be forwarded to the `Spatial Output Log`.
Have you tested if less logs are sent to the SpatialOutputLog?

Reply:
The file is still the same. The difference is that you can now choose what to include in that file depending on the chosen command line arguments.

It's true that there is a message you can send to the runtime to add a log message to the deployment logs. So yeah, we might want to choose a different name for the command line parameters. I thought to combine both user-level ops and protocol logs under the same hood of "Worker logging" since both originate from the same source. Do you have a suggestion?

Importance depends on the use case, I'd say. I tried to keep parity with what we already had, i.e. just writing logs to a file. Logs are not sent to `SpatialOutputLog`. The question is whether they should?

Reply:
Here's an extract of a file with both combined:
```
[2020-04-30 12:52:00.349] [Info ] (NetStatus) worker UnrealWorker2BDD9C5049B720BCCCE58FA912C34E64 (UDP session e0b3cb1449df8a45) negotiation successful on client
[2020-04-30 12:52:00.389] [Info ] --> (API) <ReserveEntityIds> #entities: 3000
[2020-04-30 12:52:00.389] [Info ] --> (Net) {"begin_command":{"deadline":{"deadline_ms":5000},"request":{"system_command_request":{"reserve_entity_id_request":{"num_ids_to_reserve":3000}}},"request_id":0}}
[2020-04-30 12:52:00.389] [Info ] --> (API) <CreateEntity> entity: none, #components: 6
[2020-04-30 12:52:00.390] [Info ] --> (Net) {"begin_command":{"deadline":{"deadline_ms":5000},"request":{"system_command_request":{"create_entity_request":{"template_":{"entity_state":{"50":{"1":{"1":{"1":"workerId:UnrealWorker2BDD9C5049B720BCCCE58FA912C34E64"}},"2":[{"1":54,"2":{"1":{"1":"workerId:UnrealWorker2BDD9C5049B720BCCCE58FA912C34E64"}}},{"1":53,"2":{"1":{"1":"workerId:UnrealWorker2BDD9C5049B720BCCCE58FA912C34E64"}}},{"1":50,"2":{"1":{"1":"workerId:UnrealWorker2BDD9C5049B720BCCCE58FA912C34E64"}}},{"1":58,"2":{"1":{"1":"workerId:UnrealWorker2BDD9C5049B720BCCCE58FA912C34E64"}}},{"1":9974,"2":{"1":{"1":"workerId:UnrealWorker2BDD9C5049B720BCCCE58FA912C34E64"}}},{"1":9972,"2":{"1":{"1":"workerId:UnrealWorker2BDD9C5049B720BCCCE58FA912C34E64"}}}]},"53":{"1":"WorkerEntity:UnrealWorker2BDD9C5049B720BCCCE58FA912C34E64"},"54":{"1":{"1":0,"2":0,"3":0}},"58":{"1":{"1":54,"2":{"1":[{"1":{"10":[{"8":9993},{"8":9979},{"8":9983}]},"3":"\ufffdN\ufffdN\ufffdN\ufffdM\ufffdM\ufffdM\ufffdM\ufffdN\ufffdN\ufffdN\ufffdN\ufffdM\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdN\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdS\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdV\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdW\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdX\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdY\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffd[\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdQ\ufffdQ\ufffdR\ufffdS\ufffdU\ufffdW\ufffdX\ufffdY\ufffd[\ufffdN\ufffdN\ufffdN\ufffdN\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdO\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdP\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdQ\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdR\ufffdS\ufffdS\ufffdS\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdT\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdU\ufffdW\ufffdX\ufffdX\ufffdY\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffdZ\ufffd["},{"1":{"8":60},"3":60}]}}},"9972":{"1":"652\ufffdM:\ufffdM"},"9974":{"1":"UnrealWorker2BDD9C5049B720BCCCE58FA912C34E64","2":0}}}}}},"request_id":1}}
[2020-04-30 12:52:00.469] [Info ] <-- (Net) {"end_command":{"request_id":0,"response":{"system_command_response":{"reserve_entity_id_response":{"entity_id":{"id":4}}}},"status":{"message":"","status_code":1}}}
[2020-04-30 12:52:00.491] [Info ] <-- (Net) {"end_command":{"request_id":1,"response":{"system_command_response":{"create_entity_response":{"entity_id":{"id":9005}}}},"status":{"message":"","status_code":1}}}
[2020-04-30 12:52:00.527] [Info ] <-- (Net) {"op_codes":{"code":[{"add_entity":{"entity_id":9005}},{"add_component":{"component_id":58,"entity_id":9005}},{"add_component":{"component_id":9972,"entity_id":9005}},{"add_component":{"component_id":53,"entity_id":9005}},{"add_component":{"component_id":54,"entity_id":9005}},{"add_component":{"component_id":9974,"entity_id":9005}},{"add_component":{"component_id":50,"entity_id":9005}},{"delegate_component":{"authority_epoch":0,"component_id":9974,"entity_component_update":"12-34-b2-ef-04-30-0a-2c-55-6e-72-65-61-6c-57-6f-72-6b-65-72-32-42-44-44-39-43-35-30-34-39-42-37-32-30-42-43-43-43-45-35-38-46-41-39-31-32-43-33-34-45-36-34-10-00","entity_id":9005}},{"delegate_component":{"authority_epoch":0,"component_id":58,"entity_component_update":"12-95-29-d2-03-91-29-0a-8e-29-08-36-12-89-29-0a-fe-28-0a-0f-52-03-40-89-4e-52-03-40-fb-4d-52-03-40-ff-4d-18-8c-4e-18-8f-4e-18-81-4e-18-fe-4d-18-fd-4d-18-f3-4d-18-f8-4d-18-83-4e-18-8a-4e-18-89-4e-18-88-4e-18-fb-4d-18-90-4e-18-91-4e-18-92-4e-18-93-4e-18-94-4e-18-95-4e-18-96-4e-18-97-4e-18-98-4e-18-99-4e-18-9a-4e-18-9b-4e-18-9c-4e-18-9d-4e-18-9e-4e-18-9f-4e-18-a0-4e-18-a1-4e-18-a2-4e-18-a3-4e-18-a4-4e-18-a5-4e-18-a6-4e-18-a7-4e-18-a8-4e-18-a9-4e-18-aa-4e-18-ab-4e-18-ac-4e-18-ad-4e-18-ae-4e-18-af-4e-18-b0-4e-18-b1-4e-18-b2-4e-18-b3-4e-18-b4-4e-18-b5-4e-18-b6-4e-18-b7-4e-18-b8-4e-18-b9-4e-18-ba-4e-18-bb-4e-18-bc-4e-18-bd-4e-18-be-4e-18-bf-4e-18-c0-4e-18-c1-4e-18-c2-4e-18-c3-4e-18-c4-4e-18-c5-4e-18-c6-4e-18-c7-4e-18-c8-4e-18-c9-4e-18-ca-4e-18-cb-4e-18-cc-4e-18-cd-4e-18-ce-4e-18-cf-4e-18-d0-4e-18-d1-4e-18-d2-4e-18-d3-4e-18-d4-4e-18-d5-4e-18-d6-4e-18-d7-4e-18-d8-4e-18-d9-4e-18-da-4e-18-db-4e-18-dc-4e-18-dd-4e-18-de-4e-18-df-4e-18-e0-4e-18-e1-4e-18-e2-4e-18-e3-4e-18-e4-4e-18-e5-4e-18-e6-4e-18-e7-4e-18-e8-4e-18-e9-4e-18-ea-4e-18-eb-4e-18-ec-4e-18-ed-4e-18-ee-4e-18-ef-4e-18-f0-4e-18-f1-4e-18-f2-4e-18-f3-4e-18-f4-4e-18-f6-4e-18-f7-4e-18-f8-4e-18-f9-4e-18-fa-4e-18-fc-4e-18-fe-4e-18-80-4f-18-81-4f-18-82-4f-18-83-4f-18-85-4f-18-86-4f-18-87-4f-18-88-4f-18-8a-4f-18-8b-4f-18-8d-4f-18-8e-4f-18-8f-4f-18-91-4f-18-93-4f-18-95-4f-18-96-4f-18-97-4f-18-98-4f-18-99-4f-18-9a-4f-18-9b-4f-18-9c-4f-18-9d-4f-18-9e-4f-18-9f-4f-18-a0-4f-18-a1-4f-18-a3-4f-18-a5-4f-18-a6-4f-18-a8-4f-18-a9-4f-18-aa-4f-18-ac-4f-18-ad-4f-18-ae-4f-18-af-4f-18-b0-4f-18-b1-4f-18-b2-4f-18-b4-4f-18-b5-4f-18-b6-4f-18-b8-4f-18-b9-4f-18-ba-4f-18-bb-4f-18-bc-4f-18-bd-4f-18-be-4f-18-bf-4f-18-c0-4f-18-c1-4f-18-c2-4f-18-c4-4f-18-c5-4f-18-c6-4f-18-c7-4f-18-c8-4f-18-c9-4f-18-ca-4f-18-cb-4f-18-cc-4f-18-cd-4f-18-ce-4f-18-cf-4f-18-d0-4f-18-d1-4f-18-d2-4f-18-d3-4f-18-d4-4f-18-d5-4f-18-d6-4f-18-d7-4f-18-d8-4f-18-d9-4f-18-da-4f-18-dc-4f-18-dd-4f-18-de-4f-18-df-4f-18-e1-4f-18-e2-4f-18-e3-4f-18-e4-4f-18-e5-4f-18-e6-4f-18-e7-4f-18-e8-4f-18-e9-4f-18-ea-4f-18-eb-4f-18-ec-4f-18-ee-4f-18-ef-4f-18-f0-4f-18-f1-4f-18-f2-4f-18-f3-4f-18-f4-4f-18-f6-4f-18-f7-4f-18-f8-4f-18-f9-4f-18-fa-4f-18-fb-4f-18-fc-4f-18-fd-4f-18-ff-4f-18-80-50-18-81-50-18-82-50-18-83-50-18-84-50-18-85-50-18-86-50-18-87-50-18-88-50-18-89-50-18-8a-50-18-8b-50-18-8c-50-18-8d-50-18-8e-50-18-8f-50-18-90-50-18-91-50-18-92-50-18-93-50-18-94-50-18-95-50-18-96-50-18-97-50-18-99-50-18-9a-50-18-9b-50-18-9c-50-18-9d-50-18-a0-50-18-a1-50-18-a4-50-18-a5-50-18-a8-50-18-a9-50-18-ac-50-18-ad-50-18-b0-50-18-b1-50-18-b2-50-18-b3-50-18-b4-50-18-b5-50-18-b6-50-18-b7-50-18-b8-50-18-b9-50-18-ba-50-18-bb-50-18-bc-50-18-bd-50-18-be-50-18-bf-50-18-c0-50-18-c1-50-18-c2-50-18-c3-50-18-c4-50-18-c5-50-18-c6-50-18-c7-50-18-c8-50-18-c9-50-18-ca-50-18-cb-50-18-cd-50-18-ce-50-18-d1-50-18-d3-50-18-d4-50-18-d6-50-18-d8-50-18-da-50-18-db-50-18-dc-50-18-dd-50-18-de-50-18-df-50-18-e0-50-18-e1-50-18-e2-50-18-e3-50-18-e4-50-18-e5-50-18-e6-50-18-e7-50-18-e8-50-18-e9-50-18-ea-50-18-eb-50-18-ec-50-18-ed-50-18-ee-50-18-ef-50-18-f0-50-18-f1-50-18-f2-50-18-f3-50-18-f4-50-18-f5-50-18-f6-50-18-f7-50-18-f8-50-18-f9-50-18-fa-50-18-fb-50-18-fd-50-18-fe-50-18-ff-50-18-80-51-18-81-51-18-82-51-18-83-51-18-84-51-18-85-51-18-86-51-18-87-51-18-88-51-18-89-51-18-8a-51-18-8c-51-18-8d-51-18-8f-51-18-91-51-18-93-51-18-94-51-18-95-51-18-96-51-18-97-51-18-98-51-18-9b-51-18-9c-51-18-9d-51-18-9e-51-18-9f-51-18-a0-51-18-a1-51-18-a2-51-18-a3-51-18-a4-51-18-a5-51-18-a6-51-18-a7-51-18-a8-51-18-a9-51-18-aa-51-18-ab-51-18-ac-51-18-ad-51-18-ae-51-18-b0-51-18-b1-51-18-b2-51-18-b3-51-18-b4-51-18-b5-51-18-b6-51-18-b7-51-18-b8-51-18-b9-51-18-ba-51-18-bb-51-18-bc-51-18-be-51-18-bf-51-18-c0-51-18-c1-51-18-c2-51-18-c3-51-18-c4-51-18-c6-51-18-c7-51-18-c8-51-18-c9-51-18-ca-51-18-cb-51-18-cc-51-18-ce-51-18-cf-51-18-d0-51-18-d1-51-18-d2-51-18-d3-51-18-d4-51-18-d5-51-18-d7-51-18-d9-51-18-da-51-18-db-51-18-dd-51-18-df-51-18-e1-51-18-e2-51-18-e3-51-18-e4-51-18-e5-51-18-e6-51-18-e7-51-18-e8-51-18-e9-51-18-ea-51-18-eb-51-18-ec-51-18-ed-51-18-ee-51-18-ef-51-18-f0-51-18-f1-51-18-f3-51-18-f4-51-18-f5-51-18-f6-51-18-f7-51-18-f8-51-18-f9-51-18-fa-51-18-fb-51-18-fc-51-18-fd-51-18-fe-51-18-ff-51-18-80-52-18-81-52-18-82-52-18-83-52-18-84-52-18-85-52-18-86-52-18-87-52-18-88-52-18-89-52-18-8a-52-18-8b-52-18-8c-52-18-8d-52-18-8e-52-18-8f-52-18-90-52-18-91-52-18-92-52-18-93-52-18-94-52-18-95-52-18-96-52-18-97-52-18-98-52-18-99-52-18-9a-52-18-9b-52-18-9c-52-18-9d-52-18-9e-52-18-9f-52-18-a0-52-18-a1-52-18-a2-52-18-a4-52-18-a5-52-18-a6-52-18-a7-52-18-a8-52-18-a9-52-18-aa-52-18-ab-52-18-ac-52-18-ad-52-18-ae-52-18-af-52-18-b0-52-18-b1-52-18-b2-52-18-b3-52-18-b4-52-18-b5-52-18-b6-52-18-b7-52-18-b8-52-18-b9-52-18-ba-52-18-bb-52-18-bc-52-18-bd-52-18-bf-52-18-c0-52-18-c2-52-18-c4-52-18-c6-52-18-c7-52-18-c8-52-18-c9-52-18-ca-52-18-cb-52-18-cc-52-18-cd-52-18-ce-52-18-cf-52-18-d2-52-18-d3-52-18-d4-52-18-d5-52-18-d6-52-18-d7-52-18-d8-52-18-d9-52-18-da-52-18-db-52-18-dc-52-18-dd-52-18-de-52-18-df-52-18-e0-52-18-e1-52-18-e2-52-18-e3-52-18-e4-52-18-e5-52-18-e6-52-18-e7-52-18-e8-52-18-e9-52-18-ea-52-18-eb-52-18-ec-52-18-ed-52-18-ee-52-18-ef-52-18-f0-52-18-f1-52-18-f2-52-18-f3-52-18-f4-52-18-f5-52-18-f6-52-18-f7-52-18-f8-52-18-f9-52-18-fa-52-18-fb-52-18-fc-52-18-fd-52-18-fe-52-18-ff-52-18-80-53-18-81-53-18-82-53-18-83-53-18-84-53-18-85-53-18-86-53-18-87-53-18-88-53-18-89-53-18-8a-53-18-8b-53-18-8c-53-18-8d-53-18-8e-53-18-8f-53-18-90-53-18-91-53-18-92-53-18-93-53-18-94-53-18-95-53-18-96-53-18-97-53-18-98-53-18-99-53-18-9a-53-18-9b-53-18-9c-53-18-9d-53-18-9e-53-18-9f-53-18-a0-53-18-a1-53-18-a2-53-18-a3-53-18-a4-53-18-a5-53-18-a6-53-18-a7-53-18-a8-53-18-a9-53-18-aa-53-18-ab-53-18-ac-53-18-ad-53-18-ae-53-18-af-53-18-b0-53-18-b1-53-18-b2-53-18-b3-53-18-b4-53-18-b5-53-18-b6-53-18-b7-53-18-b8-53-18-b9-53-18-ba-53-18-bb-53-18-bc-53-18-bd-53-18-be-53-18-bf-53-18-c1-53-18-c2-53-18-c3-53-18-c4-53-18-c5-53-18-c6-53-18-c7-53-18-c8-53-18-c9-53-18-cc-53-18-cd-53-18-ce-53-18-cf-53-18-d0-53-18-d1-53-18-d2-53-18-d3-53-18-d4-53-18-d6-53-18-d7-53-18-d8-53-18-d9-53-18-da-53-18-db-53-18-dc-53-18-dd-53-18-de-53-18-df-53-18-e0-53-18-e1-53-18-e2-53-18-e3-53-18-e4-53-18-e5-53-18-e6-53-18-e7-53-18-e8-53-18-e9-53-18-ea-53-18-eb-53-18-ec-53-18-ed-53-18-ee-53-18-ef-53-18-f0-53-18-f1-53-18-f2-53-18-f3-53-18-f4-53-18-f5-53-18-f6-53-18-f7-53-18-f8-53-18-f9-53-18-fa-53-18-fb-53-18-fc-53-18-fd-53-18-fe-53-18-ff-53-18-80-54-18-81-54-18-82-54-18-83-54-18-84-54-18-85-54-18-86-54-18-87-54-18-88-54-18-89-54-18-8a-54-18-8b-54-18-8c-54-18-8d-54-18-8e-54-18-8f-54-18-90-54-18-92-54-18-94-54-18-96-54-18-97-54-18-98-54-18-99-54-18-9a-54-18-9b-54-18-9c-54-18-9d-54-18-9e-54-18-9f-54-18-a0-54-18-a1-54-18-a2-54-18-a3-54-18-a4-54-18-a5-54-18-a6-54-18-a7-54-18-a8-54-18-a9-54-18-aa-54-18-ab-54-18-ac-54-18-ad-54-18-ae-54-18-af-54-18-b0-54-18-b1-54-18-b2-54-18-b3-54-18-b4-54-18-b5-54-18-b6-54-18-b7-54-18-b8-54-18-b9-54-18-ba-54-18-bb-54-18-bc-54-18-bd-54-18-be-54-18-bf-54-18-c0-54-18-c1-54-18-c2-54-18-c3-54-18-c5-54-18-c7-54-18-c9-54-18-ca-54-18-cb-54-18-cd-54-18-cf-54-18-d1-54-18-d2-54-18-d3-54-18-d4-54-18-d5-54-18-d6-54-18-d7-54-18-d9-54-18-db-54-18-dd-54-18-de-54-18-df-54-18-e0-54-18-e1-54-18-e2-54-18-e3-54-18-e4-54-18-e5-54-18-e6-54-18-e7-54-18-e8-54-18-e9-54-18-ea-54-18-eb-54-18-ec-54-18-ed-54-18-ee-54-18-ef-54-18-f0-54-18-f1-54-18-f2-54-18-f3-54-18-f4-54-18-f5-54-18-f6-54-18-f7-54-18-f8-54-18-f9-54-18-fa-54-18-fb-54-18-fc-54-18-fd-54-18-fe-54-18-ff-54-18-80-55-18-81-55-18-82-55-18-83-55-18-84-55-18-85-55-18-88-55-18-89-55-18-8a-55-18-8b-55-18-8c-55-18-8d-55-18-8e-55-18-8f-55-18-90-55-18-91-55-18-92-55-18-93-55-18-94-55-18-95-55-18-96-55-18-97-55-18-98-55-18-99-55-18-9a-55-18-9b-55-18-9c-55-18-9d-55-18-9e-55-18-9f-55-18-a0-55-18-a1-55-18-a2-55-18-a4-55-18-a6-55-18-a8-55-18-a9-55-18-aa-55-18-ab-55-18-ac-55-18-ad-55-18-ae-55-18-af-55-18-b0-55-18-b1-55-18-b2-55-18-b3-55-18-b4-55-18-b5-55-18-b6-55-18-b7-55-18-b9-55-18-bb-55-18-bd-55-18-be-55-18-bf-55-18-c0-55-18-c1-55-18-c2-55-18-c3-55-18-c4-55-18-c5-55-18-c6-55-18-c7-55-18-c8-55-18-c9-55-18-ca-55-18-cb-55-18-cc-55-18-cd-55-18-ce-55-18-cf-55-18-d0-55-18-d1-55-18-d2-55-18-d3-55-18-d4-55-18-d5-55-18-d6-55-18-d7-55-18-d8-55-18-d9-55-18-da-55-18-db-55-18-dc-55-18-dd-55-18-de-55-18-df-55-18-e0-55-18-e1-55-18-e3-55-18-e4-55-18-e5-55-18-e6-55-18-e7-55-18-e8-55-18-e9-55-18-ea-55-18-eb-55-18-ec-55-18-ed-55-18-ee-55-18-ef-55-18-f0-55-18-f1-55-18-f2-55-18-f3-55-18-f4-55-18-f5-55-18-f6-55-18-f7-55-18-f8-55-18-f9-55-18-fa-55-18-fb-55-18-fc-55-18-fd-55-18-fe-55-18-ff-55-18-80-56-18-81-56-18-82-56-18-83-56-18-84-56-18-85-56-18-86-56-18-87-56-18-88-56-18-89-56-18-8a-56-18-8b-56-18-8c-56-18-8d-56-18-8e-56-18-8f-56-18-90-56-18-91-56-18-92-56-18-93-56-18-94-56-18-95-56-18-96-56-18-97-56-18-98-56-18-99-56-18-9a-56-18-9b-56-18-9c-56-18-9d-56-18-9e-56-18-9f-56-18-a0-56-18-a1-56-18-a2-56-18-a3-56-18-a4-56-18-a5-56-18-a6-56-18-a7-56-18-a8-56-18-a9-56-18-aa-56-18-ab-56-18-ac-56-18-ad-56-18-ae-56-18-af-56-18-b0-56-18-b1-56-18-b2-56-18-b3-56-18-b4-56-18-b5-56-18-b6-56-18-b7-56-18-b8-56-18-b9-56-18-ba-56-18-bb-56-18-bc-56-18-bd-56-18-be-56-18-bf-56-18-c0-56-18-c1-56-18-c2-56-18-c3-56-18-c4-56-18-c5-56-18-c6-56-18-c7-56-18-c8-56-18-c9-56-18-ca-56-18-cb-56-18-cc-56-18-cd-56-18-ce-56-18-cf-56-18-d0-56-18-d1-56-18-d2-56-18-d3-56-18-d4-56-18-d5-56-18-d6-56-18-d7-56-18-d8-56-18-d9-56-18-da-56-18-db-56-18-dc-56-18-dd-56-18-de-56-18-df-56-18-e0-56-18-e1-56-18-e2-56-18-e3-56-18-e4-56-18-e5-56-18-e6-56-18-e7-56-18-e8-56-18-e9-56-18-ea-56-18-eb-56-18-ec-56-18-ed-56-18-ee-56-18-ef-56-18-f0-56-18-f1-56-18-f2-56-18-f3-56-18-f4-56-18-f5-56-18-f6-56-18-f7-56-18-f8-56-18-f9-56-18-fa-56-18-fb-56-18-fc-56-18-fd-56-18-fe-56-18-ff-56-18-80-57-18-81-57-18-82-57-18-83-57-18-84-57-18-85-57-18-86-57-18-87-57-18-88-57-18-89-57-18-8a-57-18-8b-57-18-8c-57-18-8d-57-18-8e-57-18-8f-57-18-90-57-18-92-57-18-94-57-18-95-57-18-96-57-18-97-57-18-98-57-18-99-57-18-9a-57-18-9b-57-18-9c-57-18-9d-57-18-9e-57-18-9f-57-18-a0-57-18-a1-57-18-a2-57-18-a3-57-18-a4-57-18-a5-57-18-a6-57-18-a7-57-18-a8-57-18-a9-57-18-aa-57-18-ab-57-18-ac-57-18-ad-57-18-ae-57-18-af-57-18-b0-57-18-b1-57-18-b2-57-18-b3-57-18-b4-57-18-b5-57-18-b6-57-18-b7-57-18-b8-57-18-b9-57-18-ba-57-18-bb-57-18-bc-57-18-bd-57-18-be-57-18-bf-57-18-c0-57-18-c1-57-18-c2-57-18-c3-57-18-c4-57-18-c5-57-18-c6-57-18-c7-57-18-c8-57-18-c9-57-18-ca-57-18-cb-57-18-cc-57-18-cd-57-18-ce-57-18-cf-57-18-d0-57-18-d1-57-18-d2-57-18-d3-57-18-d4-57-18-d5-57-18-d6-57-18-d7-57-18-d8-57-18-d9-57-18-da-57-18-db-57-18-dc-57-18-dd-57-18-de-57-18-df-57-18-e0-57-18-e1-57-18-e2-57-18-e3-57-18-e4-57-18-e5-57-18-e6-57-18-e7-57-18-e8-57-18-e9-57-18-ea-57-18-eb-57-18-ec-57-18-ed-57-18-ee-57-18-ef-57-18-f0-57-18-f1-57-18-f2-57-18-f3-57-18-f4-57-18-f5-57-18-f6-57-18-f7-57-18-f8-57-18-f9-57-18-fa-57-18-fb-57-18-fc-57-18-fd-57-18-fe-57-18-ff-57-18-80-58-18-81-58-18-82-58-18-83-58-18-84-58-18-85-58-18-86-58-18-87-58-18-88-58-18-89-58-18-8a-58-18-8b-58-18-8c-58-18-8d-58-18-8e-58-18-8f-58-18-90-58-18-91-58-18-92-58-18-93-58-18-94-58-18-95-58-18-96-58-18-97-58-18-98-58-18-99-58-18-9a-58-18-9b-58-18-9c-58-18-9d-58-18-9e-58-18-9f-58-18-a0-58-18-a1-58-18-a2-58-18-a3-58-18-a4-58-18-a5-58-18-a6-58-18-a7-58-18-a8-58-18-a9-58-18-aa-58-18-ab-58-18-ac-58-18-ad-58-18-ae-58-18-af-58-18-b0-58-18-b1-58-18-b2-58-18-b3-58-18-b4-58-18-b5-58-18-b6-58-18-b7-58-18-b8-58-18-b9-58-18-ba-58-18-bc-58-18-be-58-18-bf-58-18-c0-58-18-c1-58-18-c2-58-18-c3-58-18-c4-58-18-c6-58-18-c7-58-18-c8-58-18-c9-58-18-ca-58-18-cb-58-18-cc-58-18-cd-58-18-ce-58-18-cf-58-18-d0-58-18-d1-58-18-d2-58-18-d3-58-18-d4-58-18-d5-58-18-d6-58-18-d7-58-18-d8-58-18-d9-58-18-da-58-18-db-58-18-dc-58-18-dd-58-18-de-58-18-df-58-18-e0-58-18-e1-58-18-e2-58-18-e3-58-18-e4-58-18-e5-58-18-e6-58-18-e7-58-18-e8-58-18-e9-58-18-ea-58-18-eb-58-18-ec-58-18-ed-58-18-ee-58-18-ef-58-18-f0-58-18-f1-58-18-f2-58-18-f3-58-18-f4-58-18-f5-58-18-f6-58-18-f7-58-18-f8-58-18-f9-58-18-fa-58-18-fb-58-18-fc-58-18-fd-58-18-fe-58-18-ff-58-18-80-59-18-81-59-18-82-59-18-83-59-18-84-59-18-85-59-18-86-59-18-87-59-18-88-59-18-89-59-18-8a-59-18-8b-59-18-8c-59-18-8d-59-18-8e-59-18-8f-59-18-90-59-18-91-59-18-92-59-18-93-59-18-94-59-18-95-59-18-96-59-18-97-59-18-98-59-18-99-59-18-9a-59-18-9b-59-18-9c-59-18-9d-59-18-9e-59-18-9f-59-18-a0-59-18-a1-59-18-a2-59-18-a3-59-18-a4-59-18-a5-59-18-a6-59-18-a7-59-18-aa-59-18-ab-59-18-ac-59-18-ad-59-18-ae-59-18-af-59-18-b0-59-18-b1-59-18-b2-59-18-b3-59-18-b4-59-18-b5-59-18-b6-59-18-b7-59-18-b8-59-18-b9-59-18-ba-59-18-bb-59-18-bc-59-18-bd-59-18-be-59-18-bf-59-18-c0-59-18-c1-59-18-c2-59-18-c3-59-18-c4-59-18-c5-59-18-c6-59-18-c7-59-18-c8-59-18-c9-59-18-ca-59-18-cb-59-18-cc-59-18-cd-59-18-ce-59-18-cf-59-18-d0-59-18-d1-59-18-d2-59-18-d3-59-18-d4-59-18-d5-59-18-d6-59-18-d7-59-18-d8-59-18-d9-59-18-da-59-18-db-59-18-dc-59-18-dd-59-18-de-59-18-df-59-18-e0-59-18-e1-59-18-e2-59-18-e3-59-18-e4-59-18-e5-59-18-e6-59-18-e7-59-18-e8-59-18-e9-59-18-ea-59-18-eb-59-18-ec-59-18-ed-59-18-ee-59-18-ef-59-18-f0-59-18-f1-59-18-f2-59-18-f3-59-18-f4-59-18-f5-59-18-f6-59-18-f7-59-18-f8-59-18-f9-59-18-fa-59-18-fb-59-18-fc-59-18-fd-59-18-fe-59-18-ff-59-18-80-5a-18-81-5a-18-82-5a-18-83-5a-18-84-5a-18-85-5a-18-86-5a-18-87-5a-18-88-5a-18-89-5a-18-8a-5a-18-8b-5a-18-8c-5a-18-8d-5a-18-8e-5a-18-8f-5a-18-90-5a-18-91-5a-18-92-5a-18-93-5a-18-94-5a-18-95-5a-18-96-5a-18-97-5a-18-98-5a-18-99-5a-18-9a-5a-18-9b-5a-18-9c-5a-18-9d-5a-18-9e-5a-18-9f-5a-18-a0-5a-18-a2-5a-18-a3-5a-18-a4-5a-18-a5-5a-18-a6-5a-18-a7-5a-18-a8-5a-18-a9-5a-18-aa-5a-18-ab-5a-18-ac-5a-18-ad-5a-18-ae-5a-18-af-5a-18-b0-5a-18-b1-5a-18-b2-5a-18-b3-5a-18-b4-5a-18-b5-5a-18-b6-5a-18-b7-5a-18-b8-5a-18-b9-5a-18-ba-5a-18-bb-5a-18-bc-5a-18-bd-5a-18-be-5a-18-bf-5a-18-c0-5a-18-c1-5a-18-c2-5a-18-c4-5a-18-c6-5a-18-c8-5a-18-c9-5a-18-ca-5a-18-cc-5a-18-ce-5a-18-d0-5a-18-d2-5a-18-d4-5a-18-d6-5a-18-d8-5a-18-d9-5a-18-da-5a-18-db-5a-18-dc-5a-18-dd-5a-18-de-5a-18-df-5a-18-e0-5a-18-e1-5a-18-e2-5a-18-e3-5a-18-e4-5a-18-e5-5a-18-e6-5a-18-e7-5a-18-e8-5a-18-e9-5a-18-ea-5a-18-eb-5a-18-ec-5a-18-ed-5a-18-ee-5a-18-ef-5a-18-f0-5a-18-f1-5a-18-f2-5a-18-f3-5a-18-f4-5a-18-f5-5a-18-f6-5a-18-f7-5a-18-f8-5a-18-f9-5a-18-fa-5a-18-fb-5a-18-fc-5a-18-fd-5a-18-fe-5a-18-ff-5a-18-80-5b-18-81-5b-18-82-5b-18-83-5b-18-84-5b-18-85-5b-18-86-5b-18-87-5b-18-88-5b-18-89-5b-18-8a-5b-18-8b-5b-18-8c-5b-18-8d-5b-18-8e-5b-18-8f-5b-18-90-5b-18-91-5b-18-92-5b-18-93-5b-18-94-5b-18-95-5b-18-96-5b-18-97-5b-18-98-5b-18-99-5b-18-9a-5b-18-9b-5b-18-9c-5b-18-9d-5b-18-9e-5b-18-9f-5b-18-a0-5b-18-a1-5b-18-a2-5b-18-a3-5b-18-a4-5b-18-a5-5b-18-a6-5b-18-a7-5b-18-a8-5b-18-a9-5b-18-aa-5b-18-ab-5b-18-ac-5b-18-ad-5b-18-ae-5b-18-af-5b-18-b0-5b-18-b1-5b-18-b2-5b-18-b3-5b-18-b4-5b-18-b5-5b-18-b6-5b-18-b7-5b-18-b8-5b-18-b9-5b-18-ba-5b-18-bb-5b-18-bc-5b-18-bd-5b-18-be-5b-18-bf-5b-18-c0-5b-18-c1-5b-18-c2-5b-18-c3-5b-18-c4-5b-18-c5-5b-18-c6-5b-18-c7-5b-18-c8-5b-18-c9-5b-18-ca-5b-18-cb-5b-18-cc-5b-18-ed-5b-18-f0-5b-18-f1-5b-18-f2-5b-18-8c-4f-18-90-4f-18-92-4f-18-94-4f-18-a4-4f-18-db-4f-18-9e-50-18-a2-50-18-a6-50-18-aa-50-18-ae-50-18-cf-50-18-d2-50-18-d5-50-18-d7-50-18-d9-50-18-99-51-18-d6-51-18-d0-52-18-ca-53-18-86-55-18-91-57-18-bb-58-18-a8-59-18-ee-5b-18-f5-4e-18-fb-4e-18-fd-4e-18-ff-4e-18-84-4f-18-89-4f-18-a2-4f-18-a7-4f-18-ab-4f-18-b3-4f-18-b7-4f-18-c3-4f-18-e0-4f-18-ed-4f-18-f5-4f-18-fe-4f-18-98-50-18-9f-50-18-a3-50-18-a7-50-18-ab-50-18-af-50-18-cc-50-18-d0-50-18-fc-50-18-8b-51-18-8e-51-18-90-51-18-92-51-18-9a-51-18-af-51-18-bd-51-18-c5-51-18-cd-51-18-d8-51-18-dc-51-18-de-51-18-e0-51-18-f2-51-18-a3-52-18-be-52-18-c1-52-18-c3-52-18-c5-52-18-d1-52-18-c0-53-18-cb-53-18-d5-53-18-91-54-18-93-54-18-95-54-18-c4-54-18-c6-54-18-c8-54-18-cc-54-18-ce-54-18-d0-54-18-d8-54-18-da-54-18-dc-54-18-87-55-18-a3-55-18-a5-55-18-a7-55-18-b8-55-18-ba-55-18-bc-55-18-e2-55-18-93-57-18-bd-58-18-c5-58-18-a9-59-18-a1-5a-18-c3-5a-18-c5-5a-18-c7-5a-18-cb-5a-18-cd-5a-18-cf-5a-18-d1-5a-18-d3-5a-18-d5-5a-18-d7-5a-18-ef-5b-0a-06-0a-02-40-3c-18-3c","entity_id":9005}},{"delegate_component":{"authority_epoch":0,"component_id":50,"entity_component_update":"12-bb-03-92-03-b7-03-0a-39-0a-37-0a-35-77-6f-72-6b-65-72-49-64-3a-55-6e-72-65-61-6c-57-6f-72-6b-65-72-32-42-44-44-39-43-35-30-34-39-42-37-32-30-42-43-43-43-45-35-38-46-41-39-31-32-43-33-34-45-36-34-12-3d-08-36-12-39-0a-37-0a-35-77-6f-72-6b-65-72-49-64-3a-55-6e-72-65-61-6c-57-6f-72-6b-65-72-32-42-44-44-39-43-35-30-34-39-42-37-32-30-42-43-43-43-45-35-38-46-41-39-31-32-43-33-34-45-36-34-12-3d-08-35-12-39-0a-37-0a-35-77-6f-72-6b-65-72-49-64-3a-55-6e-72-65-61-6c-57-6f-72-6b-65-72-32-42-44-44-39-43-35-30-34-39-42-37-32-30-42-43-43-43-45-35-38-46-41-39-31-32-43-33-34-45-36-34-12-3d-08-32-12-39-0a-37-0a-35-77-6f-72-6b-65-72-49-64-3a-55-6e-72-65-61-6c-57-6f-72-6b-65-72-32-42-44-44-39-43-35-30-34-39-42-37-32-30-42-43-43-43-45-35-38-46-41-39-31-32-43-33-34-45-36-34-12-3d-08-3a-12-39-0a-37-0a-35-77-6f-72-6b-65-72-49-64-3a-55-6e-72-65-61-6c-57-6f-72-6b-65-72-32-42-44-44-39-43-35-30-34-39-42-37-32-30-42-43-43-43-45-35-38-46-41-39-31-32-43-33-34-45-36-34-12-3e-08-f6-4d-12-39-0a-37-0a-35-77-6f-72-6b-65-72-49-64-3a-55-6e-72-65-61-6c-57-6f-72-6b-65-72-32-42-44-44-39-43-35-30-34-39-42-37-32-30-42-43-43-43-45-35-38-46-41-39-31-32-43-33-34-45-36-34-12-3e-08-f4-4d-12-39-0a-37-0a-35-77-6f-72-6b-65-72-49-64-3a-55-6e-72-65-61-6c-57-6f-72-6b-65-72-32-42-44-44-39-43-35-30-34-39-42-37-32-30-42-43-43-43-45-35-38-46-41-39-31-32-43-33-34-45-36-34","entity_id":9005}},{"delegate_component":{"authority_epoch":0,"component_id":9972,"entity_component_update":"12-0e-a2-ef-04-0a-0a-08-36-35-32-f6-4d-3a-f4-4d","entity_id":9005}},{"delegate_component":{"authority_epoch":0,"component_id":54,"entity_component_update":"12-05-b2-03-02-0a-00","entity_id":9005}},{"delegate_component":{"authority_epoch":0,"component_id":53,"entity_component_update":"12-3e-aa-03-3b-0a-39-57-6f-72-6b-65-72-45-6e-74-69-74-79-3a-55-6e-72-65-61-6c-57-6f-72-6b-65-72-32-42-44-44-39-43-35-30-34-39-42-37-32-30-42-43-43-43-45-35-38-46-41-39-31-32-43-33-34-45-36-34","entity_id":9005}}]}}
[2020-04-30 12:52:00.527] [Info ] <-- (API) <AddEntity> entity: 9005
[2020-04-30 12:52:00.527] [Info ] <-- (API) <AddComponent> entity: 9005, component: 58
[2020-04-30 12:52:00.527] [Info ] <-- (API) <AddComponent> entity: 9005, component: 9972
[2020-04-30 12:52:00.527] [Info ] <-- (API) <AddComponent> entity: 9005, component: 53
[2020-04-30 12:52:00.527] [Info ] <-- (API) <AddComponent> entity: 9005, component: 54
[2020-04-30 12:52:00.527] [Info ] <-- (API) <AddComponent> entity: 9005, component: 9974
[2020-04-30 12:52:00.527] [Info ] <-- (API) <AddComponent> entity: 9005, component: 50
```
The `API` category is used for the user-level ops. `Net` and `NetStatus` are for protocol logs.

I'm very hesitant to continue to use protocol logging as an overarching term. Since 14.3.0 protocol logs are just one type of log you can generate. I'm currently working with Anne to update our docs as well (since they also don't reflect the 14.3.0 changes properly): https://docs.google.com/document/d/16JbMCVwRQh8oNW9fQ-AVf-lfcy96ZI7y3ZM13NRetHY/edit
I've made her aware that the term "Worker logging" has a meaning in the Unreal GDK so we will likely call the page "Debug worker instances using logs" instead.

How about we call it "Worker debug logs" then?

Reply:
WorkerSDKLogs?

Reply:
Changed command line parameters accordingly